main/busybox/0001-properly-fix-wget-https-support.patch

```diff
@@ -30,23 +40,29 @@ index cd92b3a28..a12c921cd 100644
  			tls_state_t *tls = new_tls_state();
  			tls->ifd = tls->ofd = network_fd;
  			tls_handshake(tls, servername);
- 			tls_run_copy_loop(tls);
+ 			tls_run_copy_loop(tls, flags);
  			exit(0);
  		} else {
--			char *argv[5];
-+			char *argv[6];
+-			char *argv[6];
++			char *argv[7];
+ 
  			xmove_fd(network_fd, 3);
  			argv[0] = (char*)"ssl_client";
- 			argv[1] = (char*)"-s3";
- 			//TODO: if (!is_ip_address(servername))...
+@@ -750,7 +749,13 @@ static void spawn_ssl_client(const char *host, int network_fd, int flags)
  			argv[2] = (char*)"-n";
  			argv[3] = servername;
--			argv[4] = NULL;
+ 			argv[4] = (flags & TLSLOOP_EXIT_ON_LOCAL_EOF ? (char*)"-e" : NULL);
+-			argv[5] = NULL;
 +			if (!ENABLE_SSL_CLIENT &&(option_mask32 & WGET_OPT_NO_CHECK_CERT)) {
```

Comment:
This doesn't work because `argv[4]` might already be null if `!(flags & TLSLOOP_EXIT_ON_LOCAL_EOF)`.